Fix for #24

TheTumultuousUnicornOfDarkness opened the pull request:
This fix for #24 is to avoid to use a temporary directory for `immodules.cache` and `loaders.cache` files.
Please try and comment.

TheTumultuousUnicornOfDarkness committed:
Do not create temp directories and files at runtime

Fix #24

TheTumultuousUnicornOfDarkness committed:
[CI] Fix build for Fedora containers

TheTumultuousUnicornOfDarkness commented on linuxdeploy-plugin-gtk.sh:
```diff
@@ -315,6 +304,7 @@ PATCH_ARRAY=(
     while IFS= read -r -d '' file; do
         # shellcheck disable=SC2016
-        patchelf --set-rpath '$ORIGIN/../../../..' "$APPDIR/$file"
+        patchelf --set-rpath '$ORIGIN' "$APPDIR/$file"
```
Not sure if still required. linuxdeploy.AppImage already to this automatically if I am correct.

TheAssassin replied:
I think the main issue with this line is that it forcefully overwrites the existing rpath. I think linuxdeploy now just appends to the rpath, in case it exists.

TheTumultuousUnicornOfDarkness replied:
Ok, I removed `patchelf` dependency in 07ebdecaff325b6bcabe28436aaaf7294c2dc327.

TheTumultuousUnicornOfDarkness committed:
Re-add gtk3_printbackendsdir

It seems it was removed in 3c78e96e87f648fb5cc3e909c3a4a1c87859f3da for no reason

TheTumultuousUnicornOfDarkness committed:
Fix cache file after update, not before

TheTumultuousUnicornOfDarkness committed:
Remove patchelf

https://github.com/linuxdeploy/linuxdeploy-plugin-gtk/pull/29#discussion_r808477037

TheTumultuousUnicornOfDarkness committed:
Properly strip immodules.cache

TheTumultuousUnicornOfDarkness commented:
I am able to run CPU-X without warnings now.

TheAssassin reviewed (approved):
Great to get rid of patchelf as a dependency, as well as getting rid of the tempdir. Thanks a ton @X0rg.

TheAssassin commented:
CC #21: patchelf is no longer needed.
